### PR TITLE
[BUGFIX] Use another alias for class "Object"

### LINKED ISSUE #1486: PHP 7.2 compatibility
I've tried to run Flux with PHP 7.2.0 but the following error occurred:

`Fatal error: Cannot use FluidTYPO3\Flux\Form\Container\Object as Object because 'Object' is a special class name in typo3conf/ext/flux/Classes/Form/AbstractFormComponent.php on line 15`

### PR DESCRIPTION
**Pull request by Lagerregal**
Fixes #1486

### CODE REVIEW

File: Classes/Form/AbstractFormComponent.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 use FluidTYPO3\Flux\Form\Container\Column;
 use FluidTYPO3\Flux\Form\Container\Container;
 use FluidTYPO3\Flux\Form\Container\Grid;
-use FluidTYPO3\Flux\Form\Container\Object;
+use FluidTYPO3\Flux\Form\Container\ObjectContainer;
```
Comment (cedricziel) [inline Classes/Form/AbstractFormComponent.php]: I would suggest using `Object as ObjectContainer` and we need the very same change in the test.

Comment (Lagerregal) [reply Classes/Form/AbstractFormComponent.php]: oh, my fault... of course!

**Comment by Lagerregal** [comment]
It seems that using aliases does not work in PHP 7.2 ... https://travis-ci.org/FluidTYPO3/flux/jobs/311287117#L589
We have to rename the class.

**Comment by cedricziel** [comment]
Yep. And then class_alias it on php versions < 7.2. Seems doable. Go ahead :)

**Comment by NamelessCoder** [comment]
Just restoring the diff to not add 7.2 tests. We will need to attack this one differently - by renaming the entire class.

**Comment by NamelessCoder** [comment]
I took the liberty of doing the renaming patch since there were a few string references to `'Object'` that would not have been so easy to locate. Thanks for opening this and doing the research, @Lagerregal!

**Comment by kthalmann** [comment]
Could you apply this fix in 8.2? I am still using flux 8.2.1 with typo3 7.6.32 and would like to use PHP 7.2

**Comment by NamelessCoder** [comment]
@kthalmann 8.2 is no longer maintained, sorry. Our minimum supported TYPO3 LTS version is 8.7.

**Pull request closed without merging**